controllers/complianceeventsapi/server.go

```diff
@@ -0,0 +1,366 @@
+package complianceeventsapi
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha1"
+	"database/sql"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/doug-martin/goqu/v9"
+	_ "github.com/doug-martin/goqu/v9/dialect/postgres" // blank import the dialect driver
+	_ "github.com/lib/pq"
+)
+
+var (
+	clusterKeyCache      sync.Map
+	parentPolicyKeyCache sync.Map
+	policyKeyCache       sync.Map
+)
+
+type databaseAPIServer struct {
```

Comment:
I think omitting "database" from the struct name makes it clearer that this isn't a database.
```suggestion
type complianceAPIServer struct {
```